tests/readBison.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+readBison() test
+--SKIPIF--
+<?php if (!extension_loaded("parle")) print "skip"; ?>
+--FILE--
+<?php 
+use Parle\Parser;
+use Parle\Lexer;
+
+$p = new Parser;
+
+$p->readBison("%%\n;start: 'a';%%\n");
+$p->build();
```

Comment:
Would it be expected, that any rules to be possibly added after `readBison()` call? If not, perhaps API wise it would be cleaner to have a static method that would look like 

```php
// Create and return the full object which is already in the finalized state
$p = Parser::fromBison(...);
``` 

Even if it should be possible to add more rules after reading the dump, perhaps semantically still it would make sense to have it in the static variant to save the line with the `new` call?

Thanks